.yarn/versions/32c88654.yml

```diff
@@ -0,0 +1,3 @@
+releases:
+  "@data-wrangling-components/react": patch
```

Comment:
I would probably call this "minor" since it involves adding new functionality to the public API.
- patch: small fix/update with no visible API changes
- minor: API additions that are not breaking changes
- major: breaking changes